build.gradle

```diff
@@ -124,9 +124,11 @@ android {
     }
 
     lintOptions {
+        checkGeneratedSources = true
```

Comment:
Do we want `checkGeneratedSources`? What is it used for?

Reply:
c&p from files: https://github.com/nextcloud/android/blob/master/build.gradle#L106

This is for e.g. dagger, lombok, etc.
Indeed I am unsure if this is good at all, I will remove it here.